docs/cryptography/caesar-cipher.md

```diff
@@ -3,4 +3,207 @@ title: Caesar Cipher
+	}
+
+	return encrypt;
+}
+
+int main() {
+	
+	string plain,encrypt;
+	cout<<"Enter the plain text: ";
+	cin>>plain;
+	
+	encrypt = encode(plain);
+	cout<<"Cipher Text: "<<encrypt<<endl;
+
+}
+```
+
+### Python
+
+```python
+# Function to decrypt a caesar ciphered message
+def decrypt(text, shift):
+    message = ""
+
+    # Looping through all the characters present in text
+    for character in range(len(text)):
+
+        # Saving current character in current_character
+        current_character = text[character]
+
+        # If current characater is space adding it directly to message
+        if current_character == ' ':
+            message += current_character
+            continue
+
+        # If current characater is an uppercase letter adding it after shifting with key value to message
+        if current_character.isupper():
+            message += chr((ord(current_character) - shift - 65) % 26 + 65)
+            continue
+
+        # If current characater is an lowercase letter adding it after shifting with key value to message
+        if current_character.islower():
+            message += chr((ord(current_character) - shift - 97) % 26 + 97)
+            continue
+
+    # Returning the message
+    return message
+
+
+# Encrypt Caesar Cipher Messages
+
+# Function to encrypt a message
+def encrypt(text, shift):
+    message = ""
+
+    # Looping through all the characters present in text
+    for character in range(len(text)):
+
+        # Saving current character in current_character
+        current_character = text[character]
+
+        # If current characater is space adding it directly to message
+        if current_character == ' ':
+            message += current_character
+            continue
+
+        # If current characater is an uppercase letter adding it after shifting with key value to message
+        if current_character.isupper():
+            message += chr((ord(current_character) + shift - 65) % 26 + 65)
+            continue
+
+        # If current characater is an lowercase letter adding it after shifting with key value to message
+        if current_character.islower():
+            message += chr((ord(current_character) + shift - 97) % 26 + 97)
+            continue
+
+    # Returning message
+    return message
+
+
+# Driver main function
+if __name__ == '__main__':
+    choice = input("Choose 1 for ENCRYPT and 2 for DECRYPT : ")
+    if int(choice) == 1:
+        print("Enter message to encrypt")
+        text = input()
+        print("Enter key value")
+        shift = int(input())
+        encrypted_message = encrypt(text, shift)
+        print("Encrypted message : " + encrypted_message)
+
+    elif int(choice) == 2:
+        print("Enter message to decrypt")
+        text = input()
+        print("Enter key value")
+        shift = int(input())
+        decrypted_message = decrypt(text, shift)
+        print("Decrypted message : " + decrypted_message)
+
+    else:
+        print("Wrong Choice")
+```
```

Comment:
While adding the articles, it would be preferred if you use the `mdx` file extension instead of `md`. `mdx` provides us to combine Markdown + React. And we will be using it for a good reason. 

First, we want to tab the codes like these: 

![image](https://user-images.githubusercontent.com/47351025/114029857-94458600-9897-11eb-99cf-a1a576301732.png)

Using standard markdown won't be beneficial for us. We can use some React Code to make it possible. Don't fret out if you don't know React. Here is an official [documentation](https://docusaurus.io/docs/markdown-features/code-blocks#multi-language-support-code-blocks) that you can refer to format the Code in separate tabs. Have a look and let me know if you are still in trouble.